docs/lsp-clients.json

```
@@ -162,6 +162,14 @@
     "lsp-install-server": "csharp-roslyn",
     "debugger": "Yes (netcoredbg)"
   },
+  {
+    "name": "lsp-c3",
```

The name looks a bit awkward. Maybe change to something else? 🤔

---

I was following the convention of `lsp-` + `language name`. but we can change...suggestions?

---

`lsp-` convention is for elisp file name. The `full-name:` should be the name of the language server, how about `c3-lsp` (from the repo name)?

---

Sure. Thanks for clarifying that. 

```suggestion
    "name": "c3-lsp",
```